modules/internal/BytesStringCommon.chpl

```diff
@@ -23,6 +23,7 @@ module BytesStringCommon {
   private use SysCTypes;
   private use Bytes;
   private use ByteBufferHelpers;
+  private use ChapelUtil;
```

Comment:
Did you add this because of a compilation error? I would expect this to be `use`d by default.

If we can do fine by removing it let's do so. If it is a compilation error, I'd like to know more about it and see what the bug there is.